buildkitd/dockerd-wrapper.sh

```diff
@@ -95,8 +95,23 @@ execute() {
         fi
     done
 
+    if [ "$EARTHLY_DOCKERD_CACHE_DATA" = "true" ]; then
+        # rename existing tags, so we can track which ones get re-tagged
+        for img in $(docker images -q); do
+            docker tag "$img" "earthly_cached_$img"
```

Comment:
I think it would be good to put the prefix in a var/const to avoid possible typos in the future.

Reply:
good plan.